docs/projects/docs-site/navbar/adding-items.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 1
```

Comment:
`sidebar_position` seems obsolete since we have a custom sidebar.js.  I think we should deprecate this next PR onwards.

Reply:
Yeah works anyways its not making any difference wether it stays or goes will update this in the next PR's